docs/index.md

```diff
@@ -43,6 +73,26 @@ core_lib:
         protocol: sqlite
   ...
 ```
+#### `your_core_lib.yaml` Explained:
+<b>your_core_lib.yaml is the setting for your entire Core-Lib library. The above example will show how to configure core-lib to connect to a database using [SQLAlchemy](https://docs.sqlalchemy.org/en/20/){:target="_blank"}:</b>
+- <b>log_queries:</b> Default `True`, [echo flag](https://docs.sqlalchemy.org/en/20/core/engines.html#more-on-the-echo-flag){:target="_blank"} is set to true. Set to false, SQL queries won’t be logged.
```

Comment:
this is not the same format. as other pages 

<img width="757" alt="image" src="https://github.com/shay-te/core-lib/assets/5166849/79798869-dc90-442e-8d17-98c8b844db9c">
 

should be 

<img width="922" alt="image" src="https://github.com/shay-te/core-lib/assets/5166849/031eb8c9-5653-4f03-8777-bf1bf9ba8f96">


do this for all.